.rhcicd/clowdapp-backend.yaml

```diff
@@ -151,9 +151,9 @@ objects:
           value: ${OB_TOKEN_USER}
         - name: OB_TOKEN_PASS
           value: ${OB_TOKEN_PASS}
-        - name: OB_MP_REST_URL
+        - name: QUARKUS_REST_CLIENT_OB_URL
           value: ${OB_MP_REST_URL}
-        - name: OB_KC_REST_URL
+        - name: QUARKUS_REST_CLIENT_KC_URL
```

Comment:
Fixed a typo in the old name here: `OB_KC_REST_URL` should have been `KC_MP_REST_URL`.

Reply:
Why do we do the `QUARKUS_REST_CLIENT_` prefixing? Is that required now?

Reply:
As discussed on Slack, yes this is required. Quarkus stopped supporting the old configuration keys since 2.4.x.